include/aws/s3/private/s3_client_impl.h

```diff
@@ -116,9 +92,14 @@ struct aws_s3_client {
      * internal ref count are both 0). */
     struct aws_ref_count internal_ref_count;
 
+    /* Client bootstrap for setting up connection managers. */
     struct aws_client_bootstrap *client_bootstrap;
 
-    struct aws_event_loop *event_loop;
+    /* Event loop on the client bootstrap ELG for processing work/dispatching requests. */
+    struct aws_event_loop *process_work_event_loop;
+
+    /* Event loop group for streaming request bodies back to the user. */
```

Comment:
This is actually from/to the user, right?

Reply:
Not really--the first event loop just handles scheduling of work.  The body streaming ELG actually pushes stuff back to the user.